skinning/json.md

```diff
@@ -0,0 +1,123 @@
+# skin.json
+This is the code that makes the skin work. For each keymode, you can see the column width, the hit position, and the column lighting tints.
+
+For each keymode, the format is as follows (using 7K as an example):
+
+```json
+"7k": {
+  "column_width": 80,
+  "hit_position": 144,
+  "colors": [
+    "#f7314f",
+    "#e85a5a",
+    "#f7314f",
+    "#dcc784",
+    "#f7314f",
+    "#e85a5a",
+    "#f7314f"
+  ]
+}
+```
+
+- `column_width` decides the width of all the columns in pixels.
+- `hit_position` decides how many pixels up from the bottom of the screen the hit position is.
+- `colors` tints the hit lighting of each individual column. The first entry will tint the lighting in the first lane from the left of the stage, the second entry will tint the second lane, and so on.
+
+This goes from 1K to 10K so all keymodes will need to be added.
+
+It's advised to keep the values the same between keymodes to avoid awkward looking transitions. These values will also scale all of the above images, so try to get the image width to match the column width in the .json file to make editing and setting the hit position easier.
+
+### Judgement Colours
+Also needed in the .json file are the judgement colours. These decide what colour the Judgement Splash is when displayed.
+
+The default colours for each judgement are as follows:
+
+|Judgement|Old Defaults|New Defaults|
```

Comment:
"Old defaults"? They always have been the "New Defaults". The "old" ones are from the example skin and never actually have been in the game.

Reply:
Should I use them in the template then?

Reply:
Maybe just remove the "Old" Column and rename "New Defaults" to "Color"